### ISSUE TITLE
Create benchmarks using bun

### ISSUE THREAD

**Issue by Meir017** [TechEmpower/FrameworkBenchmarks#7646]
the deno version - https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/TypeScript/deno/src/main.ts

FYI @Jarred-Sumner

**Comment by muuvmuuv** [comment]
Yes, would love other JavaScript and TypeScript engines compared here: deno and bun. Also zig (which bun is written in) as a language is also missing.

**Comment by jtwebman** [comment]
I built one real quick this weekend here: https://github.com/TechEmpower/FrameworkBenchmarks/pull/8407

**Comment by masfahru** [comment]
I think we should wait until [the implementation of `node:cluster` module](https://github.com/oven-sh/bun/issues/2428) is done

**Comment by ThatOneCalculator** [comment]
Bun does have mutlithreading with the `Bun.workers` API.

**Comment by chlorophant** [comment]
Does this example also load balance to each process somehow? If so, does anyone know how that mechanism works?

**Comment by jtwebman** [comment]
It doesn't. Node wrote their own into their stuff https://nodejs.org/api/cluster.html Bun.js doesn't support that yet but you probably could add a simple one https://github.com/oven-sh/bun/issues/2428 as at least then you are comparing apples to apples.

**Comment by NateBrady23** [comment]
Closing as part of bulk cleanup.
